dashboard/src/presenters/toAreaChart.ts

```diff
@@ -132,3 +133,17 @@ function createComparisonMap(
     };
   });
 }
+
+// Helper to generate padded sparkline-ready series from raw rows
+export function toSparklineSeries<K extends string>({
+  dataKey,
+  data,
+  granularity,
+  dateRange,
+}: DataToAreaChartProps<K>): Array<{ date: string } & Record<K, number>> {
+  const area = dataToAreaChart({ dataKey, data, granularity, dateRange });
+  return area.map((p) => ({
+    date: toDateTimeString(new Date(p.date)),
```

Comment:
This should probably just return the date instead of a date time string and let the client handle the conversion to the string, no?